qualtran/bloqs/mod_arithmetic/_shims.py

```diff
@@ -159,7 +159,7 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
         # return {(Toffoli(), 32 * self.n**2 * log2(self.n))}
         return {
             (_ModInvInner(n=self.n, mod=self.mod), 2 * self.n),
-            (Negate(self.n), 1),
+            (Negate(QUInt(self.n)), 1),
```

Comment:
What does this mean? Negating an unsigned type?

Reply:
Ah I'm actually not sure where the costs are coming from. @mpharrigan I think you added this?

Reply:
I think you've already answered my question about what does it mean to negate an unsigned type.  I am not worried about the exact costs here. I'll resolve the comment.